### ISSUE TITLE
Extend Tutorials to cover implementing Keyboard Shortcuts

### ISSUE THREAD

**Issue by mattyrob** [WordPress/gutenberg#20359]
The GitHub repository already contains some excellent tutorials for creating blocks and sidebar elements for the Block Editor.

Please consider extending or including implementation of Keyboard Shortcuts in the Tutorials with the proposed changing landing in WordPress 5.4

**Comment by kanlukasz** [comment]
Yes. And would be nice to see some examples there with `unregisterShortcut` option too

**Comment by ryanwelcher** [comment]
I think this is a great candidate to be added to the [Gutenberg Examples](https://github.com/WordPress/gutenberg-examples) repository as its purpose is to provide more detailed working examples than the docs can accommodate. cc @mburridge
